README.md

```diff
@@ -4,6 +4,7 @@ winsw: Windows service wrapper in less restrictive license
 [![Github All Releases](https://img.shields.io/github/downloads/kohsuke/winsw/total.svg)](https://github.com/kohsuke/winsw/releases)
 [![NuGet](https://img.shields.io/nuget/v/WinSW.svg)](https://www.nuget.org/packages/WinSW/)
 [![Build status](https://ci.appveyor.com/api/projects/status/i94752yal9iy77in?svg=true)](https://ci.appveyor.com/project/oleg-nenashev/winsw)
+[![Gitter](https://badges.gitter.im/winsw/winsw.svg)](https://gitter.im/winsw/winsw?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```

Comment:
What differences do these query strings make? Any stats for us?

Reply:
It is just Gitter's metadata for Gitlab's internal marketing analytics.  They do not share results with us